### DISCUSSION TITLE
Use different radarr connection for requests

### DISCUSSION THREAD

**Discussion by burtaverde** [sct/overseerr#372] [General]
Hello, i'm wondering if it's possible to use a different radarr connection depending on the user. In this case it's mostly since i have a directory for my stuff and another for others. I guess i could just use oversteer for others, but i was hoping to have everything go through it.

**Comment by samwiseg0** [comment]
I believe this will solve your request. 

https://github.com/sct/overseerr/issues/351

**Reply by burtaverde** [reply]
it does look like it would. Looking forward to it
